metadata.yaml

```diff
@@ -26,7 +26,7 @@ peers:
     interface: upgrade
 
 provides:
-  zookeeper:
+  database:
```

Comment:
**issue, question** uhm, I would have left as before. I mean, I don't know whether it creates problems to swap the name of the endpoint during upgrades. Would existing relations wire up automatically to the new endpoint? 

Why is it needed to change the name of the endpoint?

Also, I would be more concerned with the interface name. If we need to keep `zookeeper` we need to modify the name of the interface in the charm-relation PR from `zookeeper_client` to `zookeeper`. I don't think it should be too much of a problem, given we have good rationale

Reply:
Ah right, that's a leftover from when I was trying to follow the other interface examples to the letter. Reverting right now